.github/workflows/linux.yaml

```
@@ -46,16 +47,9 @@ jobs:
     - name: Build & Push
       run: |
         SHORT_SHA=$(echo ${{ env.SHA }} | head -c 8)
-        make image
-        docker tag simplyblock/spdkcsi:latest simplyblock/spdkcsi:$TAG
-        docker tag simplyblock/spdkcsi:latest simplyblock/spdkcsi:$TAG-$SHORT_SHA
-        docker push simplyblock/spdkcsi:$TAG
-        docker push simplyblock/spdkcsi:$TAG-$SHORT_SHA
-
-        docker tag simplyblock/spdkcsi:latest-arm64 simplyblock/spdkcsi:$TAG-arm64
-        docker push simplyblock/spdkcsi:$TAG-arm64
-        
-        if [ "${{ github.ref }}" = "refs/heads/master" ]; then
-          docker tag simplyblock/spdkcsi:$TAG simplyblock/spdkcsi:latest
-          docker push simplyblock/spdkcsi:latest
-        fi
+        make spdkcsi
```

should be built the image for arm64
```suggestion
        make spdkcsi GOARCH=arm64
```